packages/dso-toolkit/components/Componenten/form-elements/group-checkboxes.njk

```diff
@@ -15,21 +15,40 @@
   </div>
   <div class="dso-field-container">
     {% for option in options %}
-      {% render '@selectable', {
-        type: 'checkbox',
-        id: localId + '-' + loop.index0,
-        name: id,
-        value: option.value,
-        label: option.label,
-        disabled: option.disabled,
-        checked: option.checked,
-        required: option.required,
-        invalid: state == 'invalid',
-        infoText: option.infoText,
-        infoOpen: option.infoOpen,
-        infoButtonLabel: option.infoButtonLabel,
-        infoStatic: option.static
-      } %}
+      {% if option.nestedItems %}
```

Comment:
Ik begrijp deze constructie niet zo goed, kan iemand dit toelichten? Waarom is er een `else` nodig?

Reply:
Waarom is dit eigenlijk de verantwoordelijkheid van `group-checkboxes`?

Reply:
Ik heb dit zo opgezet omdat in het geval van nested checkboxes er een bepaalde constructie omheen moet:

```
<ul class="dso-selectable-nested-parent">
  <li>
    ##selectable##
  </li>
</ul>
```

Heb jij een beter alternatief?

Reply:
Nu ik hier nog eens naar kijk voelt het inderdaad wel erg vreemd. Ik mis bijvoorbeeld ook een `{% for item in items %}` constructie op dit niveau.

Reply:
Dit kan veel eenvoudiger, ik heb het `nested` stukje hier weggehaald, en de verantwoordelijkheid ligt nu bij het `selectable` component. Ook is de `ul li` constructie die ik dacht nodig te hebben weg.